en/docs/learn/api-microgateway/grouping-apis-with-labels.md

```diff
@@ -5,7 +5,7 @@ If required, you can create a [WSO2 API Microgateway](https://wso2.com/api-manag
 <html>
 <div class="admonition note">
 <p class="admonition-title">Note</p>
-<p>For more information on WSO2 API Microgateway, see <a href="https://docs.wso2.com/display/MG300/API+Microgateway+Documentation">API Microgateway Documentation</a>.</p>
+<p>For more information on WSO2 API Microgateway, see <a href="https://docs.wso2.com/display/MG310/API+Microgateway+Documentation">API Microgateway Documentation</a>.</p>
```

Comment:
let's change this to MG320 docs

Reply:
Fixed with: https://github.com/wso2/docs-apim/pull/1652/commits/6c61de05ac8318ecbdb32680e08454ff772195d0